site/content/docs/5.2/components/navbar.md

```diff
@@ -356,7 +360,7 @@ Navbar themes are easier than ever thanks to Bootstrap's combination of Sass and
     </div>
   </nav>
 
-  <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+  <nav class="navbar navbar-expand-lg bg-primary" data-bs-theme="dark">
```

Comment:
I've spotted few discrepancies here.

Before

![Screenshot 2022-11-25 at 08 51 13](https://user-images.githubusercontent.com/17381666/203928871-41afc86c-f59d-4e90-ae6a-3298923f3bd9.png)

Now

![Screenshot 2022-11-25 at 08 50 18](https://user-images.githubusercontent.com/17381666/203928929-64e01fd5-21d0-432c-9432-67381574809c.png)

* The gray text ("Features", etc.) is not very well contrasted
* The search input border color shouldn't be black

In dark mode, this search input seems off with a black background. Note that before dark mode, even for the dark navbar, this input background stayed dark.


![Screenshot 2022-11-25 at 08 53 44](https://user-images.githubusercontent.com/17381666/203929584-82cf6c8c-a184-420c-b0fc-39f86d1b1c3e.png)